public/controllers/agent/components/register-agent.js

```diff
@@ -515,8 +515,7 @@ export const RegisterAgent = withErrorBoundary(
                   color="warning"
                   title={
                     <>
-                      Running this command on a host with an agent already installed upgrades the
-                      agent package without enrolling the agent. To enroll it, see the{' '}
+                      Running this command will not enroll the agent if it is doing an upgrade. See{' '}
                       <EuiLink href="https://documentation.wazuh.com/current/user-manual/registering/index.html">
```

Comment:
There is a missign requirement on #4036 

> Also, the link to the documentation should open a new tab in the browser.

Add `target="_blank"` to this link, as seen [here](https://codesandbox.io/s/fcgt4y?file=/demo.js:198-213).